.github/workflows/manual-e2e.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         with:
           go-version: "1.20" # The Go version to download (if necessary) and use.
       - name: E2E changeover test
-        run: go run ./tests/e2e/... --tc changeover
+        run: go run ./tests/e2e/... --tc changeover::changeover
```

Comment:
> Also we should document the required format for each flag in a readme if that's not already done

You can see it by running --help, do you think that is good enough?

Reply:
> From description
> 
> > for test-file, specifying the test runner separately is necessary
> 
> It seems to me like specifying testrunner is mostly relevant to the `--test-file` flag. Is it neccessary to also require that callers specify the testrunner when using `--tc`?
> 
> Asking because the cli for e2e tests is becoming quite verbose to run a simple test from the manual (golang defined) test cases

Okay, I can see that. I'm between making it optional and removing it for the test cases, and I think for now I will remove it for --tc. Can be added back later if we need it.